generator/internal/mixin_utils_test.cc

```diff
@@ -219,28 +222,39 @@ TEST_F(MixinUtilsTest, GetMixinMethods) {
             "google.cloud.location.Locations.GetLocation");
```

Comment:
Also, we should use the GMock container matchers when possible. e.g.

```cc
EXPECT_THAT(mixin_methods, ElementsAre(
    Pair("key1", value1_matcher),
    Pair("key2", value2_matcher),
    Pair("key3", value3_matcher)));
```

https://google.github.io/googletest/gmock_cook_book.html#matching-containers

Reply:
Or if we change it to a vector, it would just be: `ElementsAre(value1_matcher, value2_matcher, value3_matcher)`

Reply:
Also above (GitHub doesn't let me comment on the exact line), we should use `ASSERT_THAT` if the test cannot progress past a failed expectation. e.g. `ASSERT_THAT(service, NotNull());`

Reply:
Thanks! Done.